src/main/java/de/zalando/zmon/scheduler/ng/entities/EntityRepository.java

```diff
@@ -81,11 +81,10 @@ private void createAutoCompleteData() {
             ObjectMapper mapper = new ObjectMapper();
             String v = mapper.writeValueAsString(typeMap);
             Jedis jedis = redisPool.getResource();
-
             try {
```

Comment:
you can use try-with-resource here, i.e. `try (Jedis jedis = redisPool.getResource()) {...}` --- automatically closing